### DISCUSSION TITLE
A discusions tab?

### DISCUSSION THREAD

**Discussion by ldijkman** [DanielBaulig/esphome-web-app#44] [General]
Hello

add a discussions tab to this github page

like your idea
i do not want to build a complete Home assistant network, thats to much
not tested it yet

what i see on the screenshots is link buttons that not have a human friendly name
thats not attractive, user friendly

maybe i could USE /  LINK it in my project 
for simple homecontrol visual timeslots timed switch

https://ldijkman.github.io/async-esp-fs-webserver/

![Screenshot from 2024-02-11 08-49-58](https://github.com/DanielBaulig/esphome-web-app/assets/45427770/5120f7a3-4e22-4ac6-bb7f-558647585d8c)

**Comment by DanielBaulig** [comment]
Great idea! Just added the discussions tab. Wasn't even aware of that Github feature.

https://github.com/DanielBaulig/esphome-web-app/discussions

The names of buttons and other entities are pulled from the "name" property of the ESPHome entity. 

ESPWA requires the ESPHome `web_server:` component to communicate with the ESP device. More precisely, it requires the [Web Server REST API](https://esphome.io/web-api/index.html). If your project were to implement this API, then ESPHome Web App could be used in combination with that, too!

**Comment by ldijkman** [comment]
hello

do not know much about home assistant
- maybe a visit device page button?

but i think your app 
could be usefull for people 
that want to do some automation
simpler

![Screenshot from 2024-03-02 12-23-09](https://github.com/DanielBaulig/esphome-web-app/assets/45427770/8aa42d84-1a3f-4127-85d5-5decf6c4bb32)

![Screenshot from 2024-03-02 12-58-55](https://github.com/DanielBaulig/esphome-web-app/assets/45427770/e259b101-abc4-4b61-9cfa-c37983d686c9)

**Reply by DanielBaulig** [reply]
The "Add via Wi-Fi" button says "Visit device" if ESPWA thinks the device cannot be added to the ESPWA dashboard.
